contracts/sol6/mock/MaliciousKyberNetwork.sol

```diff
@@ -111,23 +111,14 @@ contract MaliciousKyberNetwork is KyberNetwork {
         return (destAmount - myFeeWei);
     }
 
-    /* solhint-enable function-max-lines */
-
-    /// @notice use token address ETH_TOKEN_ADDRESS for ether
-    /// @dev do one trade with a reserve
-    /// @param src Src token
-    /// @param amount amount of src tokens
-    /// @param dest   Destination token
-    /// @param destAddress Address to send tokens to
-    /// @return true if trade is successful
     function doReserveTrades(
         IERC20 src,
         uint256 amount,
         IERC20 dest,
         address payable destAddress,
         TradeData memory tData,
         uint256 expectedDestAmount
-    ) internal override returns (bool) {
```

Comment:
Should remove amount here?

Reply:
malicious network is used to send a bit less dest amount. add see the proxy can identify and revert.

so this function is doing it well. don't think we should re write it.